editor/src/clj/editor/properties_view.clj

```diff
@@ -97,18 +97,33 @@
     (ui/auto-commit! update-fn)
     (select-all-on-click!)))
 
+(defn set-text-field-icon!
+  [text-field image-url]
+  (let [style (str "-fx-background-image: url('" image-url "');"
+                   "-fx-background-repeat: no-repeat;"
+                   "-fx-background-position: right 6px center;")]
+    (.setStyle text-field style)))
+
+(defn script-property-type->prop-icon-png [script-property-type]
+  (case script-property-type
+    :script-property-type-number   "icons/16/Icons_34-Gamepad.png"
+    :script-property-type-hash     "icons/16/Icons_35-Inputbinding.png"
+    :script-property-type-url      "icons/16/Icons_36-Texture.png"))
```

Comment:
For now we need distinguish between 3 properties, later we will be able to add more if we want so.